src/components/Hero.jsx

```diff
@@ -15,10 +15,11 @@ export default class Hero extends Component {
                     <h1 className="h1-title">Inspire the Future&nbsp;of&nbsp;Tech</h1>
                     <h2 className="h2-title">Inspire the Future&nbsp;of&nbsp;Tech</h2>
                     <p>Hosted virtually April 10-11, 2021</p>
+                    <p>10:00AM - 1:30PM</p>
                     <p>University of California, Los Angeles</p>
                     <div className="hero-cta">
-                        <Button variant="info" href={`#${HomePageSections.GET_INVOLVED.name}`} onClick={this._trackGetInvolvedClick}>
-                            Get Involved
+                        <Button variant="info" href={`/register`}>
```

Comment:
I think it would be a good idea to track how often this is being clicked (via Google Analytics). You can pass on `onClick` to this button and create a `_trackRegisterClick()`. See `_trackGetInvolvedClick()` as an example.